R/rd-include-rmd.R

```diff
@@ -45,9 +45,10 @@ roxy_tag_rd.roxy_tag_includeRmd <- function(x, base_path, env) {
 
   rmarkdown::render(
     rmd_path,
-    output_format = rmarkdown::github_document(html_preview = FALSE),
-    output_options = list(
-      if (packageVersion("rmarkdown") >= "2.12") math_method = NULL
+    output_format = "github_document",
+    output_options = c(
+      list(html_preview = FALSE),
+      if (packageVersion("rmarkdown") >= "2.12") list(math_method = NULL)
```

Comment:
This could also be done probably by creating the format with option using a `do.call(rmarkdown::github_document, ...)` 
I can change if that better fits the usual practice or if it seems clearer.